### PR TITLE
Improve hubspot documentation

### PR DESCRIPTION
**Pull request by girarda**

### CODE REVIEW

File: docs/integrations/sources/hubspot.md [hunk 1]
```diff
@@ -64,18 +64,18 @@ Depending on the type of engagement, different properties will be set for that o
 #### Requirements
 
 * HubSpot Account
-* Api credentials
-* If using Oauth, [scopes](https://legacydocs.hubspot.com/docs/methods/oauth2/initiate-oauth-integration#scopes) enabled for the streams you want to sync
+* API or OAuth2.0 credentials
+* If using OAuth, [scopes](https://legacydocs.hubspot.com/docs/methods/oauth2/initiate-oauth-integration#scopes) enabled for the streams you want to sync
 
 {% hint style="info" %} HubSpot's API will [rate limit](https://developers.hubspot.com/docs/api/usage-details) the amount of records you can sync daily, so make sure that you are on the appropriate plan if you are planning on syncing more than 250,000 records per day. {% endhint %}
```
Comment (sherifnada) [inline docs/integrations/sources/hubspot.md:70]:
```suggestion
{% hint style="info" %}
HubSpot's API will [rate limit](https://developers.hubspot.com/docs/api/usage-details) the amount of records you can sync daily, so make sure that you are on the appropriate plan if you are planning on syncing more than 250,000 records per day.
{% endhint %}
```

File: docs/integrations/sources/hubspot.md [hunk 2]
```diff
@@ -64,18 +64,18 @@ Depending on the type of engagement, different properties will be set for that o
 #### Requirements
```
Comment (sherifnada) [inline docs/integrations/sources/hubspot.md:64]: Can you split the getting started section and have one specifically for Airbyte cloud, and one specifically for OSS? The OSS section can be exactly the same as this, but Cloud should always recommend using oauth

**Review by sherifnada — APPROVED** [review]
small changes requested but LGTm

File: docs/integrations/sources/hubspot.md [hunk 3]
```diff
@@ -59,23 +59,35 @@ Depending on the type of engagement, different properties will be set for that o
...
+{% hint style="info" %}
+HubSpot's API will [rate limit](https://developers.hubspot.com/docs/api/usage-details) the amount of records you can sync daily, so make sure that you are on the appropriate plan if you are planning on syncing more than 250,000 records per day.
+{% endhint %}
 
-This connector supports only authentication with API Key. To obtain API key for the account go to settings -&gt; integrations \(under the account banner\) -&gt; api key. If you already have an api key you can use that. Otherwise generated a new one. See [docs](https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key) for more details.
+To obtain the API key for the account, go to settings -&gt; integrations \(under the account banner\) -&gt; api key. If you already have an API key you can use that. Otherwise, generate a new one. See [docs](https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key) for more details.
```
Comment (misteryeo) [inline docs/integrations/sources/hubspot.md:74]: Should `api key` here be in caps?

File: docs/integrations/sources/hubspot.md [hunk 4]
```diff
@@ -59,23 +59,35 @@ Depending on the type of engagement, different properties will be set for that o
 
 **Note**: HubSpot API currently only supports `quotes` endpoint using API Key, using Oauth it is impossible to access this stream (as reported by [community.hubspot.com](https://community.hubspot.com/t5/APIs-Integrations/Help-with-using-Feedback-CRM-API-and-Quotes-CRM-API/m-p/449104/highlight/true#M44411)).
 
-## Getting Started \(Airbyte Open-Source / Airbyte Cloud\)
+## Getting Started
```
Comment (misteryeo) [inline docs/integrations/sources/hubspot.md:62]: In our input configuration, we link to this section for the Client ID, Client Secret, and Refresh Token - but we don't list those as requirements here nor do we explicitly mention how to retrieve these fields. Can we please add them?

File: docs/integrations/sources/hubspot.md [hunk 5]
```diff
@@ -84,12 +96,12 @@ Example of the output message when trying to read `workflows` stream with missin
...
-        "message": 'Stream `workflows` cannot be procced. This hapikey (EXAMPLE_API_KEY) does not have proper permissions! (requires any of [automation-access])'
+        "message": 'Stream `workflows` cannot be proceed. This API key (EXAMPLE_API_KEY) does not have proper permissions! (requires any of [automation-access])'
     }
 }
 ```
 
-### Required scopes
+## Required scopes
 
 If you are using Oauth, most of the streams require the appropriate [scopes](https://legacydocs.hubspot.com/docs/methods/oauth2/initiate-oauth-integration#scopes) enabled for the API account.
```
Comment (misteryeo) [inline docs/integrations/sources/hubspot.md:106]: Can we make sure that all instances of `OAuth` are correctly capitalized for both 'O' and 'A'?

**Review by misteryeo — CHANGES_REQUESTED** [review]
Added some comments!

File: docs/integrations/sources/hubspot.md (hunk 4 above)
Comment (girarda) [reply docs/integrations/sources/hubspot.md:62]: Yes, added a link to oauth quickstart https://github.com/airbytehq/airbyte/pull/11218/files#diff-5faad53afb0b7795a4e1bd2b62eda2424f5756d8f08c2514db84be2977533ccdR82 and https://github.com/airbytehq/airbyte/pull/11218/files#diff-5faad53afb0b7795a4e1bd2b62eda2424f5756d8f08c2514db84be2977533ccdR96

File: docs/integrations/sources/hubspot.md [hunk 6]
```diff
@@ -57,25 +57,49 @@ Depending on the type of engagement, different properties will be set for that o
...
+### Requirements \(Airbyte Open-Source\)
 
 * HubSpot Account
-* Api credentials
-* If using Oauth, [scopes](https://legacydocs.hubspot.com/docs/methods/oauth2/initiate-oauth-integration#scopes) enabled for the streams you want to sync
+* API or OAuth2.0 Credentials (See below)
+
+#### Using API Credentials
+
+* API Key
```
Comment (girarda) [inline docs/integrations/sources/hubspot.md:71]: @misteryeo I'm not sure what the right capitalization for API Key should be as we're a little inconsistent across integrations.

I settled for `API Key` in this file. let me know if you prefer something else.

File: docs/integrations/sources/hubspot.md [hunk 7]
```diff
@@ -57,25 +57,53 @@ Depending on the type of engagement, different properties will be set for that o
...
+
+{% hint style="info" %}
+HubSpot's API will [rate limit](https://developers.hubspot.com/docs/api/usage-details) the amount of records you can sync daily, so make sure that you are on the appropriate plan if you are planning on syncing more than 250,000 records per day.
+{% endhint %}
+
+### Requirements \(Airbyte Cloud\)
+
+* HubSpot Account
+* Client ID
+* Client Secret
```
Comment (sherifnada) [inline docs/integrations/sources/hubspot.md:92]: they don't need any of this oauth stuff in Cloud, they can just click the oauth button. See [this example](https://docs.airbyte.com/integrations/sources/google-ads#getting-started-airbyte-cloud). 

Also could you put Cloud first, for consistency?

**Review by sherifnada — CHANGES_REQUESTED** [review]

File: docs/integrations/sources/hubspot.md (hunk 7 above)
Comment (girarda) [reply docs/integrations/sources/hubspot.md]: Replaced with same steps as in the google ads example and moved cloud before OSS

**Review by misteryeo — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

**Pull request merged**